common/src/main/java/com/wynntils/utils/mc/ComponentUtils.java

```diff
@@ -168,4 +176,42 @@ public static MutableComponent makeObfuscated(
         }
         return newName;
     }
+
+    /**
+     * Adds a Wynntils chat banner to the left side of the provided text in the style of Wynncraft 2.1 chat banners.
+     * The formatting of the provided text is preserved.
+     *
+     * @param formattedText the formatted text to add the Wynntils chat banner to
+     * @return a {@code Component} holding the formatted text with the Wynntils chat banner added
+     */
+    public static Component addWynntilsBanner(FormattedText formattedText) {
+        // - 2 to have room for the banner character and a space on the left
```

Comment:
What is this -2 referring to?

Reply:
Sorry that was probably left over from an earlier revision.